genie-web/src/main/java/com/netflix/genie/web/controllers/GenieExceptionMapper.java

```diff
@@ -105,6 +108,8 @@ public void handleGenieException(
             this.serverUnavailableRate.increment();
         } else if (e instanceof GenieTimeoutException) {
             this.timeoutRate.increment();
+        } else if (e instanceof GenieUserLimitExceededException) {
+            this.userLimitExceededRate.increment();
```

Comment:
Done (also tagged what kind of limit was exceeded, even if right now we have a single one)